lightning/src/lib.rs

```diff
@@ -143,10 +144,12 @@ mod prelude {
 	pub use alloc::string::ToString;
 }
 
+/// Sync compat for std/no_std
 #[cfg(feature = "std")]
-mod sync {
+pub mod sync {
```

Comment:
Hmm, I feel like maybe its simpler to just copy this to `lightning-invoice`? Its kinda awkward to expose it unless we need compatibility. If we really want to expose it we should `#[doc(hidden)]` it and maybe make it `_sync` instead of documenting it and including it in our API.

Reply:
done